Sources/PrimerSDK/Classes/User Interface/TokenizationViewModels/ApplePayTokenizationViewModel.swift

```diff
@@ -351,7 +443,46 @@ extension ApplePayTokenizationViewModel {
         }
     }
 
-    internal func createOrderItemsFromClientSessionAndApplePayOptions(_ clientSession: ClientSession.APIResponse, applePayOptions: ApplePayOptions?) throws -> [ApplePayOrderItem] {
+    private func updateShippingDetailsViaClientSessionActionIfNeeded(address: ClientSession.Address?,
```

Comment:
Can/should this live in the client session actions module with the shipping method selector? or maybe call some separate helpers? I like what you did with the shipping method bits, seems a shame to have this in the TVM for that reason.

Reply:
Yes, great shout 🙌